pkg/scheduler/objects/queue_test.go

```diff
@@ -1052,60 +1052,58 @@ func TestIsEmpty(t *testing.T) {
 
 func TestGetOutstandingRequestMax(t *testing.T) {
 	// queue structure:
-	// root
+	// root (max.cpu = 5)
```

Comment:
This is bit confusing especially with "root max is irrelevant for the calculation" comment. My understanding is, root max res has been set based on the actual usage just to satisfy root Q fitIn operation.

Reply:
not really confusing: it is important to know what the root max is. What can be accommodated by the root max is much lower than the outstanding requests list returns clearly showing that it is irrelevant. That is also why I did not set some strange high value but linked it to the fake allocation size.